### PR TITLE
Next head server side2

### PR DESCRIPTION
**Pull request by Edswor**
This fix the SSR so the card and deck pages have the head tags correctly loaded (so Discord and others should preview the content correctly).

My only concern is that I changed some cookie reading in init-apollo.js but I have been unable to test if it still works.

### CODE REVIEW

**Comment by alexmiddeleer** [comment]
Hey Edswor, I'm getting these errors on your branch on the deck-builder page after logging in. If you're having trouble logging in on your side, try clearing your site data - if you were logged in before and restart the server it gets messed up.

image
image

**Comment by Edswor** [comment]
> Hey Edswor, I'm getting these errors on your branch on the deck-builder page after logging in. If you're having trouble logging in on your side, try clearing your site data - if you were logged in before and restart the server it gets messed up.
> 

It should be fixed now. Can you try again?. Thanks.

**Comment by lily-pytel** [comment]
@alexmiddeleer what am I testing exactly?

**Comment by alexmiddeleer** [comment]
> @alexmiddeleer what am I testing exactly?

Just a quick regression test (I spent like 5 minutes, including nuking my Dockers + logging in). And checking that view page source (or if you want to be fancy, installing an opengraph testing browser extension) reports the opengraph descriptions for decks correctly.

**Comment by alexmiddeleer** [comment]
@Edswor Mind taking a peek at https://github.com/mythgard-hub/mythgard-hub/issues/231? I think this might have caused it.

File: next/pages/_app.js [hunk 1]
```diff
@@ -1,10 +1,8 @@
-import App, { Container } from 'next/app';
+import App from 'next/app';
 import React from 'react';
 import Router from 'next/router';
 import withApolloClient from '../components/with-apollo-client';
-import { ApolloProvider } from 'react-apollo';
```
Comment (lily-pytel) [inline next/pages/_app.js]: @alexmiddeleer @Edswor this one we do still need since we still use it

**Comment by lily-pytel** [comment]
@alexmiddeleer Sorry, forgot to check that page (I always forget about that page). This should fix the issue https://github.com/mythgard-hub/mythgard-hub/pull/232

**Pull request merged**